### ISSUE TITLE
File->Open Dataset menu option is not functional

### ISSUE THREAD

**Issue by jewettaij** [DiamondLightSource/SuRVoS#40]
For the record, the "File"->"Open Dataset" menu option does not seem to do anything yet.
(Fortunately, it looks like users can get around it by clicking in the field below "Input Dataset" in the upper-right-hand corner.)

**Comment by srikanthnagella** [comment]
File->Open Dataset on menu opens the window with in the workspace that you have seen where you could select the input dataset and output. It's not like the traditional UI file browser where you can select the file.

**Comment by jewettaij** [comment]
I see.  The first time I used SuRVoS, I clicked on the big blue "Open Dataset" box in the lower-left-hand corner, which took me to the workspace you are talking about.  Then, when I subsequently clicked on "File"->"Open Dataset" it did not do anything because I was already in that workspace.  Perhaps to avoid confusion, it would be better if the the "File"->"Open Dataset" menu option does both tasks: 1) switches to the workspace, and 2) brings up the file selection dialogue box.
Either way, most users will probably figure it out.
